content/en/news/releases/1.10.x/announcing-1.10/upgrade-notes/index.md

```diff
@@ -0,0 +1,83 @@
+The notes also mention changes which preserve backwards compatibility while introducing new behavior.
+Changes are only included if the new behavior would be unexpected to a user of Istio 1.9.
+
+## Inbound Forwarding Configuration
+
+The behavior of inbound forwarding will be modified in the near future. While this change is not enabled
+by default in Istio 1.10, it can be enabled today by configuring the `PILOT_ENABLE_INBOUND_PASSTHROUGH=true` environment
+variable in Istiod.
+
+Previously, requests would be forwarded to `localhost`. This leads to two important differences compared to running applications
+without Istio:
+
+* Applications that bind to `localhost` will be exposed to external pods.
+* Applications that bind to `<POD_IP>` will not be exposed to external pods.
+
+The latter is a common source of friction when adopting Istio, in particular with stateful services where this is common.
+
+The new behavior instead forwards the request as is. This matches the behavior a user would see without Istio installed.
+However, as a result, applications that have come to rely on `localhost` being exposed externally by Istio may stop working.
+
+To help detect these situations, we have added a check to find pods that will be impacted. You can run the istioctl
+experimental precheck command to get a report of any pods binding to lo on a port exposed in a Service. This command is
+available in Istio 1.10+. Without action, these ports will no longer be accessible upon upgrade.
+
+{{< text bash >}}
+$ istioctl experimental precheck
+Error [IST0143] (Pod echo-local-849647c5bd-g9wxf.default) Port 443 is exposed in a Service but listens on localhost. It will not be exposed to other pods.
+Error [IST0143] (Pod echo-local-849647c5bd-g9wxf.default) Port 7070 is exposed in a Service but listens on localhost. It will not be exposed to other pods.
+Error: Issues found when checking the cluster. Istio may not be safe to install or upgrade.
+See https://istio.io/latest/docs/reference/config/analysis for more information about causes and resolutions.
+{{< /text >}}
+
+Regardless of Istio version, the behavior can be explicitly controlled by the `Sidecar`.
+For example, to configure the 9080 port to explicitly be sent to localhost:
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1beta1
+kind: Sidecar
+metadata:
+  name: ratings
+spec:
+  workloadSelector:
+    labels:
+      app: ratings
+  ingress:
+  - port:
+      number: 9080
+      protocol: HTTP
+      name: http
+    defaultEndpoint: 127.0.0.1:9080
+{{< /text >}}
+
+## Sidecar Injector Changes
+
+The logic to determine if a pod requires sidecar injection or not has been updated to make use of
+new Kubernetes features. Previously, the webhook was triggered at a coarse grain level, selecting any
+pods in a namespace with a matching `istio-injection=enabled` label.
+
+This has two limitations:
+
+* Opting out individual pods with the `sidecar.istio.io/inject` label would still trigger the webhook,
```

Comment:
Is this an annotation (instead of label)?

Reply:
Yep, fixed, thanks